src/ui/components/map/providers/googlemapprovider.js

```diff
@@ -126,9 +126,22 @@ export default class GoogleMapProvider extends MapProvider {
     // Only here for demo purposes, so we'll fix later.
     setTimeout(() => {
       const container = DOM.query(el);
+
+      const zoomControlPosition = isRTL(this._language)
+        ? google.maps.ControlPosition.LEFT_BOTTOM
+        : google.maps.ControlPosition.RIGHT_BOTTOM;
+
```

Comment:
In theme we have `LEFT_TOP` and `RIGHT_TOP` so let's be consistent and make either top or bottom for both. Similar thing with mapbox one.

Reply:
Switched to match the theme